### DISCUSSION TITLE
[Question]What is the boot sequence delay for?

### DISCUSSION THREAD

**Discussion by dbuezas** [luc-github/ESP3D#977] [Questions]
I set it to zero so the board starts up faster. What is it good for?
<img width="327" alt="image" src="https://github.com/luc-github/ESP3D/assets/777196/6634c1b9-376e-44cc-ac0a-3bd0b56a15b9">

**Comment by luc-github** [comment]
it is usefull for boards which start slower than ESP3D, it avoid to overload their serial buffer with ESP3D commands / message. it allow to display the IP on printer screen on time, etc... it really depend on what board is connected to ESP3D

**Reply by dbuezas** [reply]
I see. My marlin has access to the reset pins of the esp so it resets it in the right moment.
Thanks!

**Reply by luc-github** [reply]
what is your board  / screen ?

**Reply by dbuezas** [reply]
Skr 3, powering an ultimaker 2 with a stock SSD1309.
Marlin needed a couple of PRs to get it working due to software i2c display, encoder lights and pt100 in the bed, and some hacks to speed up the display, but now it is a powerhouse

**Reply by luc-github** [reply]
usually Marlin take ~3s to finally start on such board depending on drivers, if your ESP can get DHCP IP withing 2s you won't see the IP message on screen if boot delay is 0 -  just be aware of that

**Reply by dbuezas** [reply]
Connection takes longer for some reason and I access it by its name http://ultimaker.local anyway. But thanks for the heads up!
